navigating-across-documents: revisions and additions 010-016

rwaldron opened the pull request:
Test | Canary | Edge 16 | Firefox Nightly | Safari Tech Preview
---- | ---- | ---- | ---- | ---- | 
form-action-javascript-url-returns-script-element.html | ✔| ✔| ✔| ✘ 
form-action-js-url-direct-call.html| ✘  | ✘  | ✘  | ✘ 
form-action-js-url-doc-write-script-then-navigate-href.html | ✘  | ✘  | ✘  | ✘ 
form-action-js-url-doc-writes-script-element.html | ✘  | ✘  | ✘  | ✘ 
form-action-js-url-doc-writes-script-href-cancels-response-delayed.html | ✘  | ✘  | ✘  | ✘ 
form-action-js-url-evaluated-in-target-nested-browsing-context.html| ✘  | ✔| ✔| ✘ 
form-action-js-url-executes-async.html | ✘  | ✘  | ✘  | ✘ 
form-action-js-url-returns-string-with-script-href-cancels-response-delayed.html | ✘  | ✔| ✔| ✔
href-js-url-evaluated-in-target-nested-browsing-context.html| ✘  | ✔| ✔| ✘ 
return-false-no-navigation.html| ✔| ✔| ✔| ✔
set-iframe-location-then-navigate-to-href-response-delayed.html| ✔| ✔| ✔| ✘ 
set-iframe-location-then-navigate-to-href.html | ✔| ✔| ✔| ✘ 
set-iframe-src-then-navigate-to-href-response-delayed.html | ✘  | ✔| ✔| ✘ 
set-iframe-src-then-navigate-to-href.html| ✘  | ✔| ✔| ✘ 

<!-- Reviewable:start -->

<!-- Reviewable:end -->

w3c-bots commented:
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21436)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/311564913)

w3c-bots commented:
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/22586)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/327289075)

rwaldron committed:
navigating-across-documents: revisions and explanations for 010-015, plus new tests

commit ad4a48b4b9b718d454eb0f7cf758a6d7a2826c1d
Author: Rick Waldron <waldron.rick@gmail.com>
Date:   Mon Jan 15 14:23:53 2018 -0500

    Minor nits: use a string for js: url return; file name typo fix;

commit 083c1ee6cbd0e2954669a5cc26f3354016a631cf
Author: Rick Waldron <waldron.rick@gmail.com>
Date:   Mon Jan 15 13:47:28 2018 -0500

    Replacement: 008.html will be replaced by this test.

commit 6c173357d7fd8e74d35e29e32ea4a4cba02faeab
Author: Rick Waldron <waldron.rick@gmail.com>
Date:   Mon Jan 15 13:18:37 2018 -0500

    Major refactor: simplification, eliminate async false positives, consistent test + test-runner.html, convert comment blocks into test names and assertion descriptions

    - Eliminating use of postMessage as a test signaling mechanism serves to "reveal" async false positives.
    - Implement "*-runner.html" machinery for all tests that use an iframe as a "target nested browsing context", this will prevent tests from producing blank screens when run in webkit (where javascript: url code executions ignore target)
    - Compute "*-runner.html" for each test file, instead of maintaining the file name directly in each test file.

commit 43a1f833f447c43067af4c9afabcd00f750cfd4e
Author: Rick Waldron <waldron.rick@gmail.com>
Date:   Wed Jan 10 10:52:13 2018 -0500

    Minor fixes

commit 884ada3d511ec366f92a072ced0bf0cfdd364dea
Author: Rick Waldron <waldron.rick@gmail.com>
Date:   Thu Nov 16 16:31:33 2017 -0500

    navigating-across-documents: additional revisions, and additions 010-017

commit 2826fc1ea5ed49eff59d9eed6c910eaf66b19127
Author: Rick Waldron <waldron.rick@gmail.com>
Date:   Wed Nov 1 15:47:32 2017 -0400

    navigating-across-documents: revisions and explanations for 010-*, 011, 012-*, 013, 014 & 015

rwaldron committed:
rename tests to remove irrelevant number from file names

rwaldron commented:
- Rebased
- Test results table in the PR description has been updated to reflect the current status of of the results  
- Fixed the lint failure

cc @bzbarsky @zcorpan @annevk
